agent-core/src/main/java/com/newrelic/agent/android/Agent.java

```diff
@@ -17,6 +17,7 @@
 public class Agent {
     public static final String VERSION = "#VERSION#";
     public static final String MONO_INSTRUMENTATION_FLAG = "#MONO_INSTRUMENTATION_FLAG#";
+    public static final String UNITY_INSTRUMENTATION_FLAG = "#UNITY_INSTRUMENTATION_FLAG#";
```

Comment:
How is "_Unity Instrumented_" different than Mono? Is Mono still used? If not remove, or reprovision that? What other agent would use either flag?

Reply:
I still don't see an answer for this. The argument for using both is we have another hybrid target that is also Mono-based. MONO_INSTRUMENTATION_FLAG was added explicitly for Unity (not Xamarin), which is why it's redundant.

Also, it doesn't appear the build files were modified for `UNITY_INSTRUMENTATION_FLAG`, so simply adding another flag isn't enough.

Reply:
@cthomas-newrelic you are right, those changes are not required.